newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -85,7 +85,8 @@ const StringSelectorEditor = ({
 }: StringSelectorEditorProps) => {
   let array = [];
   try {
-    array = JSON.parse(extraInfo);
+    if (extraInfo !== '') array = JSON.parse(extraInfo);
```

Comment:
The if is kind of redundant as the try catch already handles it, doesn't it?

Reply:
Yes but the `catch` will display an error in the console, which should be for the developer a sign that something went wrong somewhere before (when using the software normally, there should be no error logged in the console).
As having extraInfo being empty is not an exceptional case (but on the contrary something very common when we're just setting up the array for the first time), it's better to handle it outside of the catch.